benchmarks/2019.json

```diff
@@ -3982,7 +4036,7 @@
     "benchmarkYear": 2017,
     "performanceYear": 2019,
     "submissionMethod": "registry",
-    "isToppedOut": true,
+    "isToppedOut": false,
```

Comment:
Is this okay?

Reply:
hmm it looks like it is topped out though based on the deciles. I don't know why it flipped. I think this should still say true.